aws/elb.go

```diff
@@ -0,0 +1,77 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/gruntwork-io/aws-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func waitUntilElbDeleted(svc *elb.ELB, input *elb.DescribeLoadBalancersInput) error {
+	for i := 0; i < 30; i++ {
+		_, err := svc.DescribeLoadBalancers(input)
+		if err != nil {
+			// an error is returned when ELB no longer exists
+			return nil
+		}
+
+		time.Sleep(1 * time.Second)
```

Comment:
Log here (debug) that you're waiting for the EBS volume to be deleted so it's clear what's going on.